### PR TITLE
[FEAT] Adding new CSS for header images of apps. 

### PR DESCRIPTION
**Pull request by simonsasse**
These images are now always fully contained and not cropped so that all information on these images is displayed.

### CODE REVIEW

File: _layouts/app.html [hunk 1]
```diff
@@ -3,7 +3,11 @@
 ---
 <article class="app">
 
-  <header class="app-header">
+  <header>
+    {% if page.header.app_image %}
+<div class="app-header" style="background-image: url({{ page.header.app_image | relative_url }})">
+</div>
+  {% endif %}
```
Comment (marehr) [inline _layouts/app.html:10]:
```suggestion
  <header class="app-header">
    {% if page.header.app_image %}
    <div class="app-image" style="background-image: url({{ page.header.app_image | relative_url }})"></div>
    {% endif %}
```

File: assets/css/style.scss [hunk 2]
```diff
@@ -60,6 +60,16 @@ $news-entry-sep: 1.5em !default;
   max-height: 434px;
 }
 
+.app-header {
```
Comment (marehr) [inline assets/css/style.scss:63]:
```suggestion
.app-image {
```

File: _apps/fiona.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ layout: app
 cite: fu_mi_publications1451
 contact: Marcel Schulz
 categories: official
+header:
+  app_image: /assets/images/apps/fiona.jpg
```
Comment (marehr) [inline _apps/fiona.md:8]: This one has weird watermarks on it. It it even a real photo?

**Review by marehr — COMMENTED** [review]
Thank you, I haven't added an app-image, because I had the feeling that most pictures were just the headline as images. I don't know if we should waste band-width for this.

Especially Fiona and Anise don't have any good images, but some others have. So we should add images.

**Comment by simonsasse** [comment]
@marehr is right, most of the images hold no information. Still, there is the possibility to add a header image, if it Is needed. :)

File: _layouts/app.html [hunk 4]
```diff
@@ -4,6 +4,10 @@
 <article class="app">
 
   <header class="app-header">
+    {% if page.header.app_image %}
+<div class="app-image" style="background-image: url({{ page.header.app_image | relative_url }})">
+</div>
+  {% endif %}
```
Comment (marehr) [inline _layouts/app.html:10]:
```suggestion
    {% if page.header.app_image %}
    <div class="app-image" style="background-image: url({{ page.header.app_image | relative_url }})"></div>
    {% endif %}
```

**Pull request merged**